projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/facet-list/facet-list.component.ts

```diff
@@ -109,4 +130,55 @@ export class FacetListComponent {
   block(event?: MouseEvent) {
     event?.stopPropagation();
   }
+
+  protected subscribeToFocusUpdates(): void {
```

Comment:
I think this method could be something more meaningful like:
```suggestion
  protected enableFocusHandlingOnFacetListChanges(): void {
```